src/Compilers/CSharp/Test/Semantic/Semantics/NullableReferenceTypesTests.cs

```diff
@@ -66628,6 +66605,7 @@ public void ObliviousTypeParameter_01()
 #pragma warning disable {(int)ErrorCode.WRN_UnreferencedField}
 #pragma warning disable {(int)ErrorCode.WRN_UnreferencedFieldAssg}
 #pragma warning disable {(int)ErrorCode.WRN_UnreferencedVarAssg}
+#pragma warning disable {(int)ErrorCode.WRN_UnassignedInternalField}
```

Comment:
Addition of this line is likely to cause an unnecessary merge conflict with [https://github.com/dotnet/roslyn/pull/34797](https://github.com/dotnet/roslyn/pull/34797) because it chnages line numbers in the baseline below. Consider adding this pragma immediately after the ```$@"``` instead. Like this:
```
$@"#pragma warning disable {(int)ErrorCode.WRN_UnassignedInternalField}
```

This should preserve the line numbers.